_sub/monitoring/helm-grafana-agent/main.tf

```diff
@@ -10,25 +10,28 @@ resource "helm_release" "grafana_agent" {
 
   values = [
     templatefile("${path.module}/values/values.yaml", {
-      cluster_name                  = var.cluster_name,
-      api_token                     = var.api_token,
-      prometheus_url                = var.prometheus_url,
-      prometheus_username           = var.prometheus_username,
-      loki_url                      = var.loki_url,
-      loki_username                 = var.loki_username,
-      tempo_url                     = var.tempo_url,
-      tempo_username                = var.tempo_username,
-      traces_enabled                = var.traces_enabled,
-      open_cost_enabled             = var.open_cost_enabled
-      enable_side_by_side           = var.enable_side_by_side,
-      agent_resource_memory_limit   = var.agent_resource_memory_limit,
-      agent_resource_memory_request = var.agent_resource_memory_request,
-      tolerations                   = var.tolerations,
-      affinity                      = var.affinity,
-      agent_replicas                = var.agent_replicas,
-      storage_enabled               = var.storage_enabled,
-      storage_class                 = var.storage_class,
-      storage_size                  = var.storage_size
+      cluster_name                           = var.cluster_name,
+      api_token                              = var.api_token,
+      prometheus_url                         = var.prometheus_url,
+      prometheus_username                    = var.prometheus_username,
+      loki_url                               = var.loki_url,
+      loki_username                          = var.loki_username,
+      tempo_url                              = var.tempo_url,
+      tempo_username                         = var.tempo_username,
+      traces_enabled                         = var.traces_enabled,
+      open_cost_enabled                      = var.open_cost_enabled
+      enable_side_by_side                    = var.enable_side_by_side,
+      agent_resource_memory_limit            = var.agent_resource_memory_limit,
+      agent_resource_memory_request          = var.agent_resource_memory_request,
+      tolerations                            = var.tolerations,
+      affinity                               = var.affinity,
+      agent_replicas                         = var.agent_replicas,
+      storage_enabled                        = var.storage_enabled,
+      storage_class                          = var.storage_class,
+      storage_size                           = var.storage_size,
+      prometheus_node_exporter_priorityclass = var.priority_class,
```

Comment:
Is it on purpose that both  prometheus_node_exporter_priorityclass and  kube_state_metrics_priorityclass are using the same var.priority_class?

Reply:
well this is how it is in the current kube-prom-stack if you look under _sub/compute/helm-kube-prometheus-stack/main.tf. I don't have the background info for our choice on priority classes